### ISSUE TITLE
Series names beyond last visible series are inaccessible

### ISSUE THREAD

**Issue by ddraayer** [danvk/dygraphs#637]
If I have series A, B, and C, and only A is visible, I would like to be able to make B visible via a g.setVisibility call. To get the index for series B requires a g.indexFromSetName("B") call, but this yields undefined. It appears that indexFromSetName returns undefined for any series name beyond the last visible series. See https://jsfiddle.net/eM2Mg/6479/ for an example.

I took only a quick peek at the code, but I suspect that in the definition of Dygraph.prototype.drawGraph_ in dygraph.js, a reference to `points.length` should be `labels.length` instead (?):
```javascript
for (var i = 1; i < points.length; i++) {
    this.setIndexByName_[labels[i]] = i;
    if (!this.visibility()[i - 1]) continue;
    this.layout_.addDataset(labels[i], points[i]);
    this.datasetIndex_[i] = dataIdx++;
}
```

**Comment by danvk** [comment]
That is odd. My understanding was that `g.getValue(0, indexFromSetName(x)) == x` should be true for every column.

The solution would be to fix `indexFromSetName`. Not generating intermediate data for invisible series is a nice optimization that I wouldn't want to drop.

**Comment by danvk** [comment]
Fixed by #771
